src/Kdyby/Translation/DI/TranslationExtension.php

```diff
@@ -207,12 +232,39 @@ protected function loadConsole(array $config)
 	protected function loadDumpers()
 	{
 		$builder = $this->getContainerBuilder();
+		$config = $this->getConfig();
 
 		foreach ($this->loadFromFile(__DIR__ . '/config/dumpers.neon') as $format => $class) {
 			$builder->addDefinition($this->prefix('dumper.' . $format))
 				->setClass($class)
 				->addTag(self::DUMPER_TAG, $format);
 		}
+
+		$dumper = $config['database']['dumper'];
+		$isDumper = false;
+		if ($dumper !== NULL) {
+			$isDumper = true;
+		} else {
+			$loader = $config['database']['loader'];
+			if (in_array($loader, array_keys(self::$dbLoaders))) { //if you register doctrine of nettedb loader, dumper is also registered
+				$isDumper = true;
+				$dumper = $loader;
+			}
+		}
+		if ($isDumper) {
```

Comment:
tady tohle cely overovani mi prijde nejaky zbytecne slozity, neslo by to napsat jednodušeji?